ui-tests/yarn.lock

```diff
@@ -1,2841 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Not sure if we want to bring this yarn-lock back, or we could add it to the gitignore, it is probably fine for it to be generated every time.